test cases/frameworks/13 yelp/meson.build

```diff
@@ -1,8 +1,5 @@
 project('yelp', 'c')
 
 itstool = find_program('itstool', required: false)
-if not itstool.found()
-  error('MESON_SKIP_TEST itstool not found.')
-endif
```

Comment:
This should not be removed, I think, because its purpose is to skip this test when itstool is not installed, not error out.

If we don't skip it then before your PR `ninja` would error out when building, and after your PR, `meson setup` will fail when configuring, but both result in testsuite errors.

Reply:
Added this back, sorry for my ignorance. I thought removing this would be a good way of testing that the bug fix works.